operator/v2/k8s/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Required? We have Licenses at the root of the repository.

Reply:
deleted